index.js

```diff
@@ -78,6 +78,8 @@ var defer = typeof setImmediate === 'function'
  * @param {Boolean} [options.saveUninitialized] Save uninitialized sessions to the store
  * @param {String|Array} [options.secret] Secret for signing session ID
  * @param {Object} [options.store=MemoryStore] Session store
+ * @param {Object} [options.signature] Object that has the same API as node-cookie-signature
```

Comment:
Typo: node-cookie-signature -> cookie-signature

Also, I'm not sure we should define our API as "being the same as another module". Perhaps just spell out what we want and just say that `cookie-signature` is what is following our API, instead of the other way around.